spec/Section 2 -- Language.md

```diff
@@ -1094,3 +1095,24 @@ and operations.
 
 As future versions of GraphQL adopt new configurable execution capabilities,
 they may be exposed via directives.
+
+
+## Document Directives
+
+DocumentDirective :
+  - Directive[Const]
+
+{DocumentDirective} is special kind of {Definition} and should be handled
+on AST level by implementation in similar way like {TypeSystemDefinition}
+or {ExecutableDefinition}.
+
+Documents of any kind may contain zero or more {DocumentDirective}s.
+Same directive may be repeated multiple times at document level:
+
+```graphql example
+@import(from: "./common")
+@import(from: "./user")
+```
+
+Unknown directives may be ignored, but it is recommended for services to fail
+if directive is not recognized.
```

Comment:
@langpavel It contradicts rules for all other directives:
https://facebook.github.io/graphql/June2018/#sec-Directives-Are-Defined

Personally, I think we shouldn't have any exceptions for top-level directives.

Reply:
So, this sentence can be dropped entirely, ok?